doc/TOPP_tutorial/TOPP_scripting.doxygen

```diff
@@ -34,6 +34,267 @@
+              <TD>
+                   <b>Output</b>
+              </TD>
+        </TR>
+        <TR>
+              <TD>
+                   @b Description: <br>
+                   The RTAlignment metric checks what the retention time was before the alignment
+                   and how it is after the alignment. These two values are added to the metavalues
+                   in the PeptideIdentification.
+              </TD>
+
+              <TD>
+                   <b> Changes in files:</b> <br>
+                   Metavalues: <br>
+                   &emsp;- 'rt_align'set to retention time after alignment <br>
+                   &emsp;- 'rt_raw' set to retention time before alignment <br>
+                   <b> Other Output:</b> <br>
+                   No additional output.
+              </TD>
+        </TR>
+        <TR>
+            <TH colspan="2">
+                <br><b>TIC</b>
+            </TH>
+        </TR>
+        <TR>
+              <TD>
+                   <b>Required input data:</b> raw mzML file
+              </TD>
+              <TD>
+                   <b>Output</b>
+              </TD>
+        </TR>
+        <TR>
+              <TD>
+                   @b Description: <br>
+                   This TIC metric calculates the total ion count of an MSExperiment if a bin size in RT seconds
+                   greater than 0 is given. All MS1 abundances within a bin are summed up.
+              </TD>
+
+              <TD>
+                   <b> Changes in files:</b> <br>
+                   This metric does not change anything in the data.<br>
+                   <b> Other Output:</b> <br>
+                   Returns:<br>
+                   &emsp;- TIC chromatograms
+              </TD>
+        </TR>
+        <TR>
+            <TH colspan="2">
+                <br><b>TopNoverRT</b>
+            </TH>
+        </TR>
+        <TR>
+              <TD>
+                   <b>Required input data:</b> PostFDR FeatureXML, raw mzML file
+              </TD>
+              <TD>
+                   <b>Output</b>
+              </TD>
+        </TR>
+        <TR>
+              <TD>
+                   @b Description: <br>
+                   The TopNoverRT metric calculates the ScanEventNumber (number of the MS2 scans after the MS1 scan) and adds
+                   them as the new metavalue 'ScanEventNumber' to the PeptideIdentifications. It finds
+                   all unidentified MS2-Spectra and adds corresponding 'empty' PeptideIdentifications without sequence as
+                   placeholders to the unassigned PeptideIdentification list. Furthermore
+                   it adds the metavalue 'identified' to the PeptideIdentification.<br>
+              </TD>
+
+              <TD>
+                   <b> Changes in files:</b> <br>
+                   Metavalues: <br>
+                   &emsp;- 'ScanEventNumber' set to the calculated value <br>
+                   &emsp;- 'identified' set to '+' or '-' <br>
```

Comment:
the metric adds a few more Metavalues. They do not really fit the name, but please mention them here nontheless. We might move them to a distinct metric at some point but thats another issue.